gimme

```diff
@@ -316,13 +364,16 @@ unset GOOS
 unset GOPATH
 unset GOROOT
 
+GIMME_VERSION_PREFIX="$(_realpath "${GIMME_VERSION_PREFIX}")"
+GIMME_ENV_PREFIX="$(_realpath "${GIMME_ENV_PREFIX}")"
```

Comment:
Ahhh.  Ok.  I think I can improve on `_realpath` slightly then (although OSX support is tricky here for a clean solution -- ideally we'd just `readlink -f` but `greadlink` isn't common on OSX :skull:).